python/docs/_build/html/_sources/conduct.rst.txt

```diff
@@ -0,0 +1 @@
+.. include:: ../CONDUCT.rst
```

Comment:
does the `_build` directory have to be committed to Git for GitHub-based installation to work?